src/api/participantsRouter.ts

```diff
@@ -39,7 +50,7 @@ export const hasParticipantAccess = async (
 };
 
 participantsRouter.get('/', async (_req, res) => {
-  const participants = await Participant.query().withGraphFetched('types');
+  const participants = await Participant.query().whereNotNull('siteId').withGraphFetched('types');
```

Comment:
A get to `/participants` should return all participants - this adds an unexpected filter. We still might want to fetch all participants, including those with no site id.

Adding a URL parameter is a common way to filter a get request - so something like /participants/?withSiteId=true

Reply:
Coz if a participant is without a site id, it means those participants are not yet approved by TAM, which I think the external user should have access to, if we make this endpoint to  /participants/?withSiteId=true, that means theoretically any user is able to retrieve those un-approved participants.

My initial thought was to add a middleware to check the user permission, if a user is a TAM, this endpoint will return all participants, otherwise return the participant with site_id, but since we haven't implement that permission, so I return all participants with site_id for now